src/tests/JIT/Regression/JitBlue/Runtime_93876/Runtime_93876.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+using System.Numerics;
+using Xunit;
+
+public static class Runtime_93876
+{
+    [Fact]
+    public static void Problem()
+    {
+        Vector4 v = Mul(0, 1);
+        Assert.Equal(Vector4.One, v);
+    }
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static Vector4 Mul(float a, float b) => Vector4.Multiply(a + b, Vector4.One);
```

Comment:
What is the type of `a + b` here that causes the issue?

Reply:
Hmm, I think `MultiplyByScalar` shouldn't be handled by this path. E.g. `Vector64.Multiply(Vector64<float>.One, x)` will produce `<x, 0>` instead of `<x, x>` because of that. So this needs a bit more work.

Reply:
I pushed a change that handles `MultiplyByScalar` separately, can you take another look?